types/src/executable.rs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{access_path::AccessPath, state_store::state_key::StateKey};
+use aptos_crypto::HashValue;
+use std::sync::Arc;
+
+#[derive(PartialEq)]
+pub enum ExecutableDescriptor {
```

Comment:
See my previous comment. Should we move these types to mvhashmap for now?

Reply:
same reasoning, but in any case this will have to evolve soon.